### DISCUSSION TITLE
Custom-source Sinkhole game settings do not match that of Battleground Sinkhole settings

### DISCUSSION THREAD

**Discussion by mawilson** [BattlesnakeOfficial/feedback#209] [Bug Reports]
I was just running some custom games to test some Sinkhole-specific stuff I'd written. During [this Custom game](https://play.battlesnake.com/g/7bc8b4c3-ae34-4ae0-9998-a745d600a1c0/), I noticed that the sinkhole boundaries expanded every 10 turns (instead of 20 like in the Sinkhole Battleground), & also that the hazard damage per turn is 15 instead of 10.

Is this intentional? If so, is there a way for us to easily create a custom game whose settings mirror that of the Sinkhole battleground?

**Comment by chris-bsnake** [comment]
Good point.  I will fix both this one and the the healing pools custom game configuration as well so they match what is going on in Battlegrounds

**Comment by chris-bsnake** [comment]
Fix is deployed
